library/alloc/src/collections/btree/map.rs

```diff
@@ -462,7 +462,7 @@ impl<K: Debug + Ord, V: Debug> Debug for Entry<'_, K, V> {
 pub struct VacantEntry<'a, K: 'a, V: 'a> {
     key: K,
     handle: Handle<NodeRef<marker::Mut<'a>, K, V, marker::Leaf>, marker::Edge>,
-    length: &'a mut usize,
+    dormant_map: DormantMutRef<'a, BTreeMap<K, V>>,
 
     // Be invariant in `K` and `V`
     _marker: PhantomData<&'a mut (K, V)>,
```

Comment:
I wonder if this (and `OccupiedEntry`'s) `PhantomData` is still relevant, now that `DormantMutRef` has the same contraint, as far as I understand.

Reply:
It is probably not necessary any more but also does not hurt.